recipes/luarocks/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "lua-luarocks" %}
```

Comment:
Sorry I missed this before. The package name needs to match the directory name where the recipe is stored. Should it be `luarocks` or `lua-luarocks`? Please send a follow-up PR to correct this and ping core.